pgbouncer/tox.ini

```
@@ -6,6 +6,8 @@ envlist =
 
 [testenv]
 dd_check_style = true
+description =
+    py{27,37}-{1.7,1.8}: e2e ready
```

```suggestion
    py{27,37}: e2e ready
```

`{1.7,1.8}` is not needed, the suggestion above will have the same effect.